### PR TITLE
chore: remove latest flavor for price-feeder docker release

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Update release tags for price feeder docker release

### CODE REVIEW

File: .github/workflows/price-feeder-release.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ jobs:
           go-version: 1.19
           cache: true
           cache-dependency-path: price-feeder/go.sum
+          flavor: |
```
Comment (gsk967) [inline .github/workflows/price-feeder-release.yml:22]: We need to add this one inside the `price-feeder-docker.yml` file at docker metadata

**Review by gsk967 — CHANGES_REQUESTED** [review]
Please add 
```
flavor: |
  latest=false
```
inside the `price-feeder-docker.yml` file at docker metadata
https://github.com/docker/metadata-action#flavor-input

**Review by gsk967 — APPROVED** [review]
LGTM

File: .github/workflows/price-feeder-release.yml (hunk 1 above)
Comment (robert-zaremba) [reply .github/workflows/price-feeder-release.yml]: thanks for catching. Updated. I'm testing in a local fork and will report if everything works.

**Comment by robert-zaremba** [comment]
thanks for catching the issue.

**Pull request merged**